AddedInfoReprodNumber&TransmDyns.pathogenesis.md (#319)
greenelab/covid19-review: content/07.pathogenesis.md

#### Reproduction Number and Dynamics of Transmission

Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.

> In models the quantities of basic reproduction (or reproductive) number and the timescale of infection have a large influence on the population level dynamics of epidemics.

Reviewer: ```suggestion
The quantities of basic reproduction (or reproductive) number and the timescale of infection have a large influence on the population-level dynamics in epidemic models.
```
It reads a little weird to me to have the prepositional phrase at the beginning and then a different one at the end (I feel like I don't know what we're talking about, exactly). Is this what you meant? If this doesn't work, perhaps there's another way to streamline it?

Reviewer 2: ```suggestion
The quantities of the basic reproduction (or reproductive) number and the effective reproduction number have a large influence on disease population dynamics.
```

Reviewer 2: yes I think this sentence needs to be clarified - "in models" not needed and I don't know what is meant by "timescale of infections"

Author: I made a paraphrase from a modern infectious disease modeling textbook, I'll go back & re-check the exact wording. I think they mean for example the average time between start of primary infection and start of secondary infection -- in other words the generation time. (also I think maybe mathematically since R0 is unitless it can only theoretically describe shapes but you also need something indicating a timescale / with units of time; relatedly, you can make a simple early epidemic growing graph by raising R0 to the exponent of #generations = time divided by generation time & also the same textbook has a related equation using the SIR with simple demographics for early epidemics as I(t) ~ I(0) exp([R0-1][gamma+mu]t) where here generation time is like inverse (gamma+mu). People usually just mention R0 because it's important & probably for other reasons I don't even know of, but I was trying to give a mathematically complete picture so that it makes more sense to people outside the field.

Author: @rando2 @alavendelm @rdvelazquez  I am still learning how to use this platform & this is a bit of complicated pull request for me to do while I'm learning. I may have messed up in trying to fix this. Here's basically, what I'd like to do and would like assistance with. I went back to the exact wording of the textbook again & would like to suggest one possible way to fix this sentence which would be closer to the wording of the textbook & also include a part of the textbook that is related to the original text emphasis on R0 (& what @alavendelm was bringing back with his suggestion); however, I'm not sure about using such a strong word as "govern" which is what the textbook uses & that's why I wanted to put something weaker (like "influence" but that may be too weak; maybe we could use "describe the" rather than "govern"??).:
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@ISBN:9780691116174].
OR
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) describe the population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@ISBN:9780691116174].
Would one of you maybe insert one of these more accurate sentences as a suggestion and then I can "commit suggestion" or some other process you suggest? Thank you for allowing for a bit of messiness while I'm learning!

Reviewer 3: @yemarshall git and GitHub are not easy to learn at first. You've done great jumping in and figuring things out! 

I don't know what process you have been using for making edits so this suggestion might not be applicable but, just in case you weren't aware of it, you can use GitHub.com to make edits to this pull-request at any time. Once you opened the pull-request it is basically saying that you want to incorporate changes from your `patch-9` branch into greenelab's `master` branch... so any changes that you make to your `patch-9` branch while this pull-request is open will show up here (and likewise, any changes that you make here via the `commit suggestion` will show up in your `patch-9` branch). 

To make a change to your `patch-9` branch you can just edit this file: https://github.com/yemarshall/covid19-review/edit/patch-9/content/07.pathogenesis.md and click the green `commit changes` button at the bottom. To get to this file I clicked the blue `yemarshall:patch-9` at the top of this page and then navigated to `content` then `07.pathogenesis.md`, then clicked the pencil icon that says "edit this file" when you hover over it.

There are many ways to use git/github, what I described above isn't the best or preferred way of doing things it's just another thing to be aware of. If it helps you great, if not feel free to disregard. Thanks again for all your additions and discussions; it's been really informative for me to follow along. Happy editing!

The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].

> R~0~ is a unitless number which in models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period.

Reviewer 2: ```suggestion
R~0~ is a unitless number which is calculated from data on transmission rates (based on the number of interactions between infectious and susceptible individuals) and the infectious period.
```

Reviewer 3: Should we also mention that the likelihood of an interaction resulting in a new infection is another major determining factor?

Reviewer 3: Is R0 actually _calculated_ this way? I just assumed that infections/interaction and interactions/unit-time were too difficult to estimate to make calculating R~0~ from these feasible... If it's not calculated this way maybe replace "calculated" with "equivalent to" or something like that?

Author: This is why the original sentence as I added had the detail "in models".  At least for some pathogens/epidemics, it's easier to measure the infectious period & the R0 & then derive the transmission rate by the relation of R0 ~ transmission rate / infectious period. (I'll go through the whole thing again to try to make this pull request's components better b/c so much quick changing of math/semantic details is making it messy. I tried to add sentences that went along with the current structure without modifying other people's sentences scientifically/mathematically unless there was something really obviously unclear which there wasn't for this section or in general. I could have done it better though by inserting more of the references or holding off on more complicated topics.)

Author: After looking at this again, I saw where the ambiguity is coming from, @alavendelm @rdvelazquez. I meant "in mechanistic models" (distinguishing between models which are more derived from biological mechanisms and models which incorporate less biological mechanisms while comparing to data and sort of in between models like those that are sometimes used to derive R from early epidemic data). (Also note, in a mechanistic model, transmission rate could be broken down into smaller details such as what @rdvelazquez  mentioned, so the sentence already covers the smaller details)  Here's a suggested better sentence incorporating a few of the references as well:
"R~0~ is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (e.g., rate of infection transmitting interactions for an infectious individual) and the infectious period [@DOI:10.1126/science.abb5659][@ISBN:9780691116174]. "
If this resolves all the comments then I'd appreciate if someone can put in this better sentence.

> When R~0~ is greater than 1, there is potential for an epidemic to grow.

Reviewer 2: ```suggestion
When R~0~ is greater than 1, the number of infected individuals grows exponentially.
```

Author: I put "potential" because I meant that some areas may not have any infectious individual (or object) introduced yet but their qualities may allow for an epidemic to grow if an infectious individual(s) is introduced (may require multiple times due to stochasticity). Also I think R0 is a characteristic of the population regardless, even if R changes; meaning sometimes R0 can be greater than 1 but R isn't?  I think this is why when people talk about R0 & R it can get confusing to people outside the field -- using formulas maybe makes it easier to follow. But what you corrected I think may be the more common statement & maybe "exponential" does account for the case of the unstable zero equilibrium (since 0 growing exponentially doesn't make sense -- I'd have to look up the theoretical math definitions again!). I'm writing this sort of quickly but just wanted to include a bit of a response. This will take some time to get to clearer additions & if I add more then I will try better to stay within a clear structure.

Reviewer 2: Thanks @yemarshall I get your point, i.e. "potential" to grow to account for noisy introduction of virus & small pop sizes. however in the specific case of this covid pandemic, with many large infected populations, R>1 means almost certainly that # infected will grow, and my fear including "potential" here is that lay reader will put it in the place of "almost certainly", which is not the case.

Author: good point, I'm going to think about this a bit more too.

Reviewer: Non-expert here -- would something like "an epidemic will grow, with the potential for exponential growth" cover both concerns?

Author: @alavendelm How about: "If R~0~ > 1, a pathogen can invade." OR "If R~0~ > 1, a pathogen can invade and start an epidemic with exponential growth of the number of infected individuals." OR I even also considered removing this sentence altogether and just keeping the Rt > 1 or Rt < 1 sentence. These are closer to the definition, & they account for the case where R0>1 but Rt<1 (e.g. with interventions) (?? note that I've worked on mechanistic models & largely avoided this terminology). I understand trying to avoid misinterpretations & to make things clear, but we also need to make sure that we don't simplify so much that it can cause other misunderstanding? In the end, from what I've read for other diseases, sometimes in obvious cases people see when the science works & when it doesn't though (1) they may not consider the uncertainty & variation and (2) they sometimes forget or don't pay attention to history (unless reminded? or unless it's essential/current?) -- this is way more complicated a topic than what I know well but I do think that people like it when science/medicine/whatever works to help them in obvious way.

Reviewer 3: One other thing to consider from a non-expert perspective: The way R0 and Rt are currently described it sort of sounds like they are a property of the disease separate from the population/epidemic and that depending on "the reproduction number" the epidemic will grow or shrink or change in some way... the reproduction number could also be thought of as simply a "description" of how the epidemic is growing or shrinking or changing in some way. 

Especially when Rt is calculated from population level statistics, it's not that "when Rt is greater than 1 an epidemic has the potential to grow" it's that "Rt greater than 1, by definition, means an epidemic is growing".

Reviewer 2: Agree with comment from @rdvelazquez this is an important distinction and the latter phrasing captures it well

> Rt greater than 1, by definition, means an epidemic is growing

Reviewer 3: ```suggestion
```
What if we just delete this sentence? The sentence in line 54 seems to cover it well enough.

Reviewer 3: from @yemarshall:

> On thinking about it again, it seems my suggestion may be lax to intentionally skip on explaining what R0<1 or R0>1 b/c people see it everywhere & so it's useful to provide an explanation that's both correct and clear. Since we also include what R>1 and R<1 means then we don't have to worry as much about people misinterpreting R0<1 or R0>1. Either we put what we have, or I can put another suggestion which makes it even more clear while also being correct and including making difference between R0 & Rt even clearer if needed -- among all the discussion there are at least a couple of suggestions which cover everything well enough (& this along with other stuff i wrote address Question4 in #313 ). If needed I can also / will just settle on the final strategy I found for doing these things to reduce so much discussions -- I'll put as close as possible to textbook definitions or mathematically exact descriptions with references & which are as clear as possible.

Your reasoning for not wanting to skip explaining what R0 > 1 means makes a lot of sense. I think the explanation will be closely related to how we explain the differences between R0 and Rt... @alavendelm are you planning to address the differences between R0 and Rt in your forthcoming PR? and if so @yemarshall are you ok with deleting this sentence for now and then revisiting after @alavendelm's PR is opened?

> The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~, for example as some fraction of the population becomes infected, or as interventions are put into place.

Reviewer: ```suggestion
The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~ that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
```

> When R~0~ is greater than 1, there is potential for an epidemic to grow.

Reviewer: ```suggestion
A pathogen can invade a susceptible population only if R~0~ > 1 [@ISBN:9780691116174; @ISBN:9780199209996].
```
I believe this is what I'm supposed to do per @yemarshall request, with the assumption that @alavendelm will likely edit in the course of handling #313 !

> When R~0~ is greater than 1, there is potential for an epidemic to grow.

Reviewer: ```suggestion
A pathogen can invade a susceptible population only if R~0~ > 1 [@ISBN:9780691116174; @ISBN:9780199209996].
```

> When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that's infectious increases); when R~t~ is smaller than 1, the proportion of the population that's infectious decreases.

Reviewer: ```suggestion
When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
```

> When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that's infectious increases); when R~t~ is smaller than 1, the proportion of the population that's infectious decreases.

Reviewer 2: ```suggestion
As for R~0~, if R~t~ is greater than 1, then the number of infected individuals increases over time; for R~t~ less than 1 the number of infected individuals decreases over time. Implementing measures that decrease or maintain R~t~ below 1 is thus necessary for containing an outbreak. 
```

Reviewer 2: I think may be important to add a caveat here about placing too much emphasis on real-time estimates of Rt - the use of these to make policy decisions is worrying as I do not think the concept is well-understood, plus short-window averages are likely to contain a lot of noise! Plus testing delays, etc etc. eg. [this](https://news.sky.com/story/coronavirus-virus-could-be-wiped-out-in-london-in-weeks-as-rate-of-infection-is-slowing-say-researchers-11988579) misleading headline.

Author: I also am more experienced in mechanistic modeling rather than lots of real-time Rt estimates but this is what is reviewed so far & whoever wrote it did a good job of putting a variety of articles, so I was trying to complement this by putting clearer definitions earlier on.

Author: @alavendelm there's a very short (~3 page) science paper which may have some points slightly related at least in general  to your point of possible misinterpretations (e.g. when science and its application isn't understood/explained clearly enough or when nuances or math are skipped). Regardless, you may find it interesting. I'll add a brief summary as an issue but really the whole 3 pages are dense with interesting information. Article/Perspectives: Modeling infectious disease dynamics. The spread of the coronavirus SARS-CoV-2 has predictable features.  S. Cobey, Science 10.1126/science.abb5659 (2020)

Author: I think these sentence incorporating some of this suggestion would be good, with the 2nd suggested sentence by @alavendelm fully solidifying to the readers early in the section that the goal is to get R below 1 (which was the main thing I was going for & maybe addresses in part @rdvelazquez Issue topic on clarifying R):
"If R~t~ is greater than 1, then the number of infected individuals increases over time; for R~t~ less than 1 the number of infected individuals decreases over time.
Implementing measures that decrease or maintain R~t~ below 1 is thus necessary for containing an outbreak."
Again, it would be better at this point if someone fixed/added this so I don't make a learner's mistake on this somewhat complicated pull request.

The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because Sweden did not implement social distancing measures on the same scale as other countries).
The study concludes that while large changes in R~t~ are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R~t~ below 1.

> More generally, population level epidemic dynamics can be both observed and modelled.

Reviewer: ```suggestion
More generally, population-level epidemic dynamics can be both observed and modelled.
```

> Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to try to make predictions on possible future dynamics, such as for capacity needs or for the comparison of predicted possible outcomes of different prevention and control strategies.

Reviewer: ```suggestion
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs or the comparison of predicted outcomes among prevention and control strategies.
```

Reviewer 4: ```suggestion
Data and empirically determined biological mechanisms inform models.  We can use models to understand data and systems of interest. Models can also be useful for making predictions on future dynamics such as the estimation of capacity needs or the comparison of predicted outcomes among prevention and control strategies.
```

> Beyond model predictions related to cases and deaths, infectious disease population models can also incorporate the dynamics of finer details such as possible genetic variations in pathogens or sometimes even possible evolutionary changes in pathogens due to relatively large numbers of pathogens which can evolve relatively quickly.

Reviewer: ```suggestion
Beyond model predictions related to cases and deaths, infectious disease population models can also incorporate the dynamics of finer details such as genetic variation among pathogens or even possible evolutionary changes in pathogens, due to relatively large numbers of pathogens which can evolve relatively quickly.
```
Whoo-hoo, my evolutionary biology background is finally useful! I am not sure I understand exactly what you mean by the last part of this sentence. My suspicion is that you mean that as even if mutation rate is extremely low, as you get a larger and larger population, that number becomes non-trivial and you are likely to see mutations arise. However, in terms of evolutionary change, these mutations would need to become more frequent at the population level, which depends on a number of factors that aren't really discussed here (I'm thinking specifically of mutation-selection balance and genetic drift). I'm happy to help clarify this text if you let me know more about what you mean!

Reviewer 2: I do not really see what the point of this paragraph is? What is its central message? Not sure it belongs in this section?

Author: There were a few different points that may be would be better re-organized, I'll try to make it clearer. 1 point was to give an idea to non-epidemiologists & non-population-modelers what models do & how they interact with biological research. For the last part, I was hinting (but did not want to definitively say since it's a new infectious disease and requires quite a lot of computation) at a possible very interesting yet also looming thing to try to prevent which is that with very large numbers you can sometimes start to see pathogen evolution in populations in real time & this might for example occur in relation to therapeutics (drug resistance) or vaccines (e.g. ?immune escape? & also antigenic variation; there's starting to be some literature already on SARS-CoV-2 strains but I haven't yet gotten a chance to look at it closely so I don't know if it's related to antigens -- the idea that detailed spatial sequence data & population models may be used is very interesting, it happens for other infectious diseases though really I hope this pathogen won't get so complicated & maybe our control strategies can reduce such complications?? I really don't know but I also think it may require quite a deal of computation to answer so it may be too early to tell. That's related to what makes the science in real-time thing so unique!). Thanks for the feedback. This is working thing that'll get better with time but I thought these would be useful to have regardless.

Author: Since this is quite a lot (see comment above), I may have gotten ahead of myself because it's so interesting (yet also something to prevent). I will look up / re-read some literature mentions of this & see if I can find a more grounded description while also mentioning how this is just something that has been seen in other pathogens and we don't know how much it will be related to this pandemic.

Reviewer 3: This seems really interesting but I wonder if there's another place in the paper where it would fit better. I think I remember seeing something about a potential evolution section... @rando2 was that discussed somewhere?

Author: copying this from my summary comment below: On looking again at this last sentence I added, I think I need to re-consider whether it is too new or different or early to include this for this pathogen/epidemic or, alternatively, if I happen to find/include some solid references where they mentioned it in the background or discussion at least or if I can make a non-flimsy connection to this type of research for other pathogens/epidemics; I know there's at least some stuff related to this but I see that I didn't write the sentence carefully enough to include as is / yet, and it may have been better to just maybe put it in as an Issue (rather than pull request) as a possible interesting topic.

Reviewer: @rdvelazquez there have been several people who expressed interest in developing that section, but I haven't seen anyone come back with a PR yet! It's something that will definitely eventually be added since I am dying to look at the pangolin/cat/bat comparative genomics question!

Reviewer 4: I think there are several points evo can be inserted @rdvelazquez @rando2 @yemarshall and this discussion is a bit of a mix. For instance,

1) Evolution analysis as in phylogenetic analysis and comparative genomics to show where SARS-CoV-2 falls in the virus tree. What makes SARS-CoV-2 more affective as a strain compared to SARS-CoV-1 - i.e. why is this a pandemic and can that be answered by genomic changes?

2) Evolution of SARS-CoV-2 since it was first discovered - the answer would be not much given a short time frame but how does it compare to what we know of other viruses? other RNA viruses? Can we try to beat the clock and move ahead of virus' evolution, if any?

3) Evolution of SARS-CoV-2 as it hopped between different species. Does it overlap what we see in 1? Can we predict 1 or 2 based on what we know of other zoonotic diseases from similar transmission potentials? Is there a comparative, attenuated strain infecting other species we can use for human vaccination?

and many more... I'd recommend sprinkling them across their respective, relevant bigger topics rather than having the pan-ultimate evo section, but that's just a personal preference.

Reviewer 3: I think sprinkling the evolution discussions within their respective relevant topics could be best. If we find that we are repeating the same general evolution information (mutation rates, evidence of selection, phylogenetic reconstruction, etc.) as background in multiple places it might be worth pulling that all into one location at that point.

Reviewer 3: I would be ok with removing this sentence from this pull request for now and then another pull-request could be opened adding in a more thorough discussion of how reproduction number and evolution overlap if that's desired.

Author: I also agree with @rdvelazquez that maybe simpler to just remove (at least for now) this last sentence I had suggested as an addition. (if later it makes sense to discuss population modeling which incorporates evolution then it can be added later; but after thinking it over better I can't actually say right now that it makes sense.) Would someone remove this sentence for me while I'm still learning this platform?

Reviewer 4: ```suggestion
```

Reviewer 4: @yemarshall done
